packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -50,15 +50,15 @@ import "src/libraries/DisputeTypes.sol";
 ///         Then add a call to that function inside of `run`. Be sure to call the `save` function after each
 ///         deployment so that hardhat-deploy style artifacts can be generated using a call to `sync()`.
 contract Deploy is Deployer {
-    DeployConfig cfg;
+    DeployConfig public cfg;
 
     /// @notice The name of the script, used to ensure the right deploy artifacts
     ///         are used.
     function name() public pure override returns (string memory name_) {
         name_ = "Deploy";
     }
```

Comment:
I was going to say that we shouldn't need a public `cfg` var in a deploy script, since typically all methods and vars can be internal except for the actual `run` method.

But then I saw this public `name` method that's "used to ensure the right deploy artifacts are used", so what exactly is our approach here? (Similarly, `implSalt()` is another public method which I'd think can be internal)

Reply:
I believe that you are right, this is leftover from when the contract was deployed and not inherited

Reply:
The `name()(string)` method is used to know what directory to look for in `broadcast`. A cheatcode for reading artifacts would be useful here but I am not sure of the best API

Reply:
Created a PR to make a better comment here: https://github.com/ethereum-optimism/optimism/pull/7987